main.go

```diff
@@ -1,12 +1,22 @@
 package main
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/gittuf/gittuf/internal/cmd/root"
 )
 
 func main() {
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Printf("unexpected error: %s\n\n", fmt.Sprint(r))
+			fmt.Printf("Please consider opening an issue on https:/github.com/gittuf/gittuf/issues tagged 'bug' with the steps to reproduce this state. Thanks!\n")
```

Comment:
```suggestion
			fmt.Fprintln(os.Stderr, "unexpected error: ", r)
			fmt.Fprintln(os.Stderr, string(debug.Stack()))
			fmt.Fprintln(os.Stderr, "Please consider opening an issue on https:/github.com/gittuf/gittuf/issues tagged 'bug' with the steps to reproduce this state. Thanks!")
```

Prints the stack trace, and sends everything to stderr instead of stdout.

https://go.dev/play/p/nYM8QG_0SJb

Reply:
This is neat! `debug.PrintStack()` prints to stderr inherently so I used that instead. Thanks!